scripts/build-protos.rs

```diff
@@ -92,6 +92,26 @@ fn add_serde_attrs(c: &mut prost_build::Config) {
             "keypath_stake",
             "serde(deserialize_with = \"crate::keypath::serde_deserialize\")",
         ),
+        (
+            "shiftcrypto.bitbox02.CardanoSignTransactionRequest.Certificate.VoteDelegation.CardanoDRepType.KEY_HASH",
+            "serde(rename = \"keyHash\")",
+        ),
+        (
+            "shiftcrypto.bitbox02.CardanoSignTransactionRequest.Certificate.VoteDelegation.CardanoDRepType.SCRIPT_HASH",
+            "serde(rename = \"scriptHash\")",
+        ),
+        (
+            "shiftcrypto.bitbox02.CardanoSignTransactionRequest.Certificate.VoteDelegation.CardanoDRepType.ALWAYS_ABSTAIN",
+            "serde(rename = \"alwaysAbstain\")",
+        ),
+        (
+            "shiftcrypto.bitbox02.CardanoSignTransactionRequest.Certificate.VoteDelegation.CardanoDRepType.ALWAYS_NO_CONFIDENCE",
+            "serde(rename = \"alwaysNoConfidence\")",
+        ),
```

Comment:
These are not needed because `camelCase` is the default - the dRepType enum has this attribute already:

    #[cfg_attr(feature = "wasm", serde(rename_all = "camelCase"))]

Could you remove these?

Reply:
I think you missed `make build-protos' afterwards, as these tags are still in the generated protobuf file. Please run `make build-protos` again.